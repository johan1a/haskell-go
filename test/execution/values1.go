package main

import "fmt"

func main() {
    fmt.Println("a" + " string")

    fmt.Println(1+1)
    fmt.Println(1.2)

    fmt.Println(true && false)
    fmt.Println(true || false)
    fmt.Println(!true)
}
