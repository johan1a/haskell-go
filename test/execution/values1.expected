a string
2
false
true
false
